src/test/regress/expected/multi_task_string_size.out

```diff
@@ -0,0 +1,239 @@
+	long_column_103 int,
+	long_column_104 int,
+	long_column_105 int,
+	long_column_106 int,
+	long_column_107 int,
+	long_column_108 int,
+	long_column_109 int,
+	long_column_110 int,
+	long_column_111 int,
+	long_column_112 int,
+	long_column_113 int,
+	long_column_114 int,
+	long_column_115 int,
+	long_column_116 int,
+	long_column_117 int,
+	long_column_118 int,
+	long_column_119 int,
+	long_column_120 int,
+	long_column_121 int,
+	long_column_122 int,
+	long_column_123 int,
+	long_column_124 int,
+	long_column_125 int,
+	long_column_126 int,
+	long_column_127 int,
+	long_column_128 int,
+	long_column_129 int,
+	long_column_130 int,
+	long_column_131 int,
+	long_column_132 int,
+	long_column_133 int,
+	long_column_134 int,
+	long_column_135 int,
+	long_column_136 int,
+	long_column_137 int,
+	long_column_138 int,
+	long_column_139 int,
+	long_column_140 int,
+	long_column_141 int,
+	long_column_142 int,
+	long_column_143 int,
+	long_column_144 int,
+	long_column_145 int,
+	long_column_146 int,
+	long_column_147 int,
+	long_column_148 int,
+	long_column_149 int,
+	long_column_150 int,
+	long_column_151 int,
+	long_column_152 int,
+	long_column_153 int,
+	long_column_154 int,
+	long_column_155 int,
+	long_column_156 int,
+	long_column_157 int,
+	long_column_158 int,
+	long_column_159 int,
+	long_column_160 int,
+	long_column_161 int,
+	long_column_162 int,
+	long_column_163 int,
+	long_column_164 int,
+	long_column_165 int,
+	long_column_166 int,
+	long_column_167 int,
+	long_column_168 int,
+	long_column_169 int,
+	long_column_170 int,
+	long_column_171 int,
+	long_column_172 int,
+	long_column_173 int,
+	long_column_174 int,
+	long_column_175 int,
+	long_column_176 int,
+	long_column_177 int,
+	long_column_178 int,
+	long_column_179 int,
+	long_column_180 int,
+	long_column_181 int,
+	long_column_182 int,
+	long_column_183 int,
+	long_column_184 int,
+	long_column_185 int,
+	long_column_186 int,
+	long_column_187 int,
+	long_column_188 int,
+	long_column_189 int,
+	long_column_190 int,
+	long_column_191 int,
+	long_column_192 int,
+	long_column_193 int,
+	long_column_194 int,
+	long_column_195 int,
+	long_column_196 int,
+	long_column_197 int,
+	long_column_198 int,
+	long_column_199 int,
+	long_column_200 int
+);
+SELECT create_distributed_table('wide_table', 'long_column_001');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+SET citus.task_executor_type TO 'task-tracker';
+SHOW citus.max_task_string_size;
+ citus.max_task_string_size 
+----------------------------
+ 12288
+(1 row)
+
+-- setting can not be changed on runtime
+SET citus.max_task_string_size TO 20000;
+ERROR:  parameter "citus.max_task_string_size" cannot be changed without restarting the server
+-- error message may vary between executions
+-- hiding warning and error message
+-- no output means the query has failed
+SET client_min_messages to FATAL;
+SELECT u.* FROM wide_table u JOIN wide_table v ON (u.long_column_002 = v.long_column_003);
```

Comment:
I think the only thing we're really testing here is whether we correctly error out when the string is very long. We would perhaps just add a query with a very long string in the target list to one of the existing task-tracker tests.